webview/js/api.js

```diff
@@ -20,12 +20,12 @@ window.pywebview = {
     },
     _bridge: {
         call: function (func_name, params) {
-            if (window.external) {
-                return window.external.call(func_name, params)
-            } else if (window.qt) {
+            if (window.qt) {
```

Comment:
Why does the order of ifs matter here?

Reply:
Because in some recent versions of QT, `window.external` is present, without
pywebview creating one. If we check for it first, we will wrongly assume that it
is part of our JS API object. Please read 5883780 commit message.